vnext/ReactCommon/ReactCommon.vcxproj

```diff
@@ -163,9 +168,11 @@
       Using Strict FloatingPointModel on x64 to workaround a compiler issue, See https://github.com/microsoft/react-native-windows/issues/4122
       Using a patched copy of Yoga due to https://github.com/microsoft/react-native-windows/issues/3994
     -->
-    <ClCompile Condition="'$(Platform)' == 'x64'" Include=".\Yoga.cpp" AdditionalIncludeDirectories="$(YogaDir)\yoga;%(AdditionalIncludeDirectories)" DisableSpecificWarnings="4244;%(DisableSpecificWarnings)" FloatingPointModel="Strict" />
-    <ClCompile Condition="'$(Platform)' != 'x64'" Include=".\Yoga.cpp" AdditionalIncludeDirectories="$(YogaDir)\yoga;%(AdditionalIncludeDirectories)" DisableSpecificWarnings="4244;%(DisableSpecificWarnings)" />
+    <ClCompile Include=".\Yoga.cpp" AdditionalIncludeDirectories="$(YogaDir)\yoga;%(AdditionalIncludeDirectories)" DisableSpecificWarnings="4244;%(DisableSpecificWarnings)" />
```

Comment:
nit, can remove the .\ prefix

Reply:
I had to make the `<FloatingPointModel>Strict</FloatingPointModel>` for the whole project in order to use the PCH file. Otherwise it fails to compile. I had moved the related comment next to the property settings to indicate it.